predicators/envs/grid_row.py

```diff
@@ -226,10 +296,10 @@ def _get_tasks(self, num: int,
                 self._light: {
                     "x": len(self._cells) - 0.5,
                     "level": 0.0,
-                    "target": rng.uniform(0.5, 1.0),
+                    "target": 0.75,
                 },
                 self._door: {
-                    "x": rng.choice(range(2, len(self._cells))) - 0.5,
+                    "x": len(self._cells) // 2 + 0.5,
```

Comment:
maybe add a comment saying that the light target and door location are fixed for now, but in the future, we'd like to relax this assumption